src/components/NavigationBar/NavigationBar.js

```diff
@@ -1,14 +1,18 @@
-.NavigationBar {
+import React from "react";
+import styled from "styled-components";
+import { Profile } from "../Profile/Profile";
+
+const Nav = styled.nav`
   display: flex;
   justify-content: center;
-  position: sticky;
+  position: ${(props) => (props.sticky ? "sticky" : "static")};
```

Comment:
요런 props 같은경우는 아래와 같이 boolean 타입으로 작성하면 좋아요 ㅎㅎ
```suggestion
  position: ${(props) => (props.isSticky ? "sticky" : "static")};
```

넘겨줄때도 아래와 같겠죠?

```jsx
// true or false 값 
 <Nav isSticky={isSticky}>
```